docs/src/main/sphinx/release/release-376.md

```diff
@@ -0,0 +1,87 @@
+# Release 376 (7 Apr 2022)
+
+## General
+
+* Add table redirection awareness for `RENAME table` operations. ({issue}`11277`)
+* Deny adding column with comment if the connector does not support this feature. ({issue}`11486`)
+* Improve query plans and performance in the presence of inequality expressions. ({issue}`11518`)
+* Use "is distinct" semantics in `array_except`, `array_union`, `map_concat`,
+  `map_from_entries`, `multimap_from_entries`, and `multimap_agg`. ({issue}`560`)
+* Fix failure of `DISTINCT .. LIMIT` operator when input data is dictionary
+  encoded. ({issue}`11837`, {issue}`11776`)
+* Fix returning of invalid results for distinct aggregation when input data is
+  dictionary encoded. ({issue}`11837`)
+* Fix operator type exception that occurred when index join was used during
+  query execution. ({issue}`11758`)
+* Fix stage memory statistics reporting for queries running with `retry-policy`
+  set to `TASK`. ({issue}`11801`)
+
+## Security
+
+* Add support to use two-way TLS/SSL with LDAP authentication. ({issue}`11070`)
+* Fix failures in information schema role tables for catalogs using system roles. ({issue}`11694`)
+
+## JDBC driver
+
+* Remove `user` property requirement. ({issue}`11350`)
+* Add support for `ResultSet.getAsciiStream` and `ResultSet.getBinaryStream`. ({issue}`11753`)
```

Comment:
`ResultSet.getAsciiStream()` and `ResultSet.getBinaryStream()` to indicate they are methods.